src/Resources/ResourceManager/Utilities/ProcessInvoker.cs

```diff
@@ -99,16 +99,20 @@ public ProcessOutput Invoke(ProcessInput input)
                 }
             }
 
+            var stderr = new StringBuilder();
+            proc.ErrorDataReceived += (sender, e) =>  stderr.AppendLine(e.Data);
             proc.Start();
-            var stdout = proc.StandardOutput.ReadToEnd();
-            var stderr = proc.StandardError.ReadToEnd();
+    
+            // To avoid deadlocks, use an asynchronous read operation on at least one of the streams.  
```

Comment:
This implementation was taken from the official docs, which mention deadlocks:
https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.process.standardoutput